channels/whatsapp/index.rst

```diff
@@ -0,0 +1,247 @@
+
+.. warning::
+   Please be aware that the usage of **WhatsApp's cloud API** is mandatory to
+   connect WhatsApp to Zammad. You won't be able
+   to connect to WhatsApp in Zammad if you just use the WhatsApp Business *App*.
+   Have a look
+   `here <https://developers.facebook.com/docs/whatsapp/cloud-api?locale=en_US>`_
+   for further information about the Cloud API.
+
+   There may be costs for you from Meta/WhatsApp side. 1000 service
+   conversations per month are free. However, you should have a look on their
+   `pricing page <https://developers.facebook.com/docs/whatsapp/pricing>`_ to
+   be safe from surprises.
+
+   Be aware that if you go through the whole process, it is very time consuming.
+   You should have some time to follow our preparation guide.
+
+.. _prerequisites:
+
+Prerequisites
+-------------
+
+You can find a guide for the required steps on our
+:doc:`sub-page <preparation>`.
+It covers the following steps:
+
+- Create a Meta business account
+- Create a Meta developer account
+- Create a Meta developer app
+- Add WhatsApp to your app
+- Create a system user
+- Configure your WhatsApp business platform
+- Assign a not yet used phone number
+
+Additionally, you have to meet the following requirements:
+
+- Make sure your system is accessible from the internet because WhatsApp sends
+  messages via webhooks to your system.
+- Make sure to have properly setup your
+  :doc:`FQDN in Settings > System </settings/system/base>` (for the webhook
+  Meta will fire for new messages).
+
+Limitations
+-----------
+
+There are a number of limitations for this channel. Read them carefully.
+
+24 hours communication time window
+   If a customer gets in touch with you via WhatsApp, a 24 hour window is
+   opened. In this time window you can answer the question of
+   the customer. If the customer isn't replying, the conversation ends
+   after 24 hours. You can't communicate with this customer via WhatsApp due to
+   their policy until a new conversation is started by your customer.
+
+   If the ticket can not be closed and to keep the conversation ongoing, we
+   suggest to create a dedicated trigger for the WhatsApp channel. For example
+   you could send a auto-generated message to the customer after 20 hours and
+   ask to reply to your message to keep the conversation open.
+
+Customer initiated communication only
+   The WhatsApp channel only supports customer initiated communication.
+   For business initiated communication, it would be necessary to create a
+   message template which has to be verified by Meta first.
+
+Supported media types
+   Currently, only text messages, documents and videos are supported.
+
+One attachment per article
+   Currently, it's only possible to attach one file per outgoing message.
+
+File size for attachments
+   There are different maximum file sizes for different media types. To pick
+   the most important ones:
+
+   - audio: 16 MB
+   - document: 100 MB
+   - image: 50MB
```

Comment:
We should add here the rest of the supported media type limits (as outlined on [the linked page](https://developers.facebook.com/docs/whatsapp/cloud-api/reference/media#supported-media-types)).